fiaas_deploy_daemon/config.py

```diff
@@ -345,8 +347,16 @@ def _parse_args(self, args):
             help=(
                 "Disable service links in the podspec. Recommended "
                 " for new installations. Existing installations may need to skip this."
-                ),
-            action="store_false"
+            ),
+            action="store_false",
+        )
+        parser.add_argument(
+            "--dns-search-domains",
+            help="list of extra dns search domains",
+            default=[],
+            action="append",
+            type=str,
+            dest="search_domains",
```

Comment:
nitpick: maybe consider keeping the same name for the config attribute and the flag to make it easier to find in the code later